contracts/weight-strategies/YearnVaultInstantRebindStrategy.sol

```diff
@@ -426,7 +431,7 @@ contract YearnVaultInstantRebindStrategy is SinglePoolManagement, WeightValueCha
         (mem.ycrvBalance, mem.crvActual, mem.usdcBefore) = _vaultToUsdc(cfg.token, mem.crvToken, vc);
```

Comment:
`mem.usdcBefore` is calculated inside `_vaultToUsdc` function as  `usdcBefore = USDC.balanceOf(address(this));` and the value is not used inside `_vaultToUsdc`.
Consider calculating "usdcBefore" inside `_instantRebind` - it should be in the same function as "usdcAfter" (calculated at L442)